src/actors/runtime/AbstractActor.ts

```diff
@@ -54,12 +54,12 @@ export default abstract class AbstractActor {
   constructor(daprClient: DaprClient, id: ActorId) {
     this.daprClient = daprClient;
     this.actorClient = new ActorClient(
-      daprClient.getDaprHost(),
-      daprClient.getDaprPort(),
-      daprClient.getCommunicationProtocol(),
-      daprClient.getOptions(),
+      daprClient.options.daprHost,
```

Comment:
Is options always set? Wouldn't `daprClient?.options?.` be better?

Reply:
See below.